http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java

```diff
@@ -407,6 +409,25 @@ public HttpClientConfig setKerberosRemoteServiceName(String serviceName)
         return this;
     }
 
+    public enum NameType
```

Comment:
@electrum is there a common place where I can put this enum? It's defined in presto-main too, but doesn't seem appropriate to include `http-client` in krb5 authenticator.

Reply:
No, it's better to have two copies

Reply:
Let's name this `KerberosNameType` so that it's un-ambiguous and make it a top level class. It would be fine to reuse it in `presto-main` since it already depends on `http-client`.